src/components/ItemTile/ItemTile.js

```diff
@@ -0,0 +1,57 @@
+import { graphql, useStaticQuery } from 'gatsby';
+import { GatsbyImage, getImage } from 'gatsby-plugin-image';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import { container, content, image, text, title } from './ItemTile.module.scss';
+
+export const ItemTile = ({ projectTitle, projectText, projectImage }) => {
+  const data = useStaticQuery(graphql`
+    query {
+      allFile(
+        filter: {
+          sourceInstanceName: { eq: "images" }
+          internal: { mediaType: { regex: "/image/" } }
+        }
+      ) {
+        nodes {
+          relativePath
+          childImageSharp {
+            gatsbyImageData(
+              layout: CONSTRAINED
+              placeholder: BLURRED
+              formats: [AUTO, WEBP, AVIF]
+            )
+          }
+        }
+      }
+    }
+  `);
+
+  const matchedImage = data.allFile.nodes.find(
+    (node) => node.relativePath === projectImage
+  );
+
+  if (!matchedImage) {
```

Comment:
Jeśli jakiegoś zdjęcia nie będzie to nie może nam zwracać całego pustego komponentu. Proponuję w tedy wyświetlić ten komponent tylko z tytułem i opisem.
Co myślisz?

Reply:
> Jeśli jakiegoś zdjęcia nie będzie to nie może nam zwracać całego pustego komponentu. Proponuję w tedy wyświetlić ten komponent tylko z tytułem i opisem. Co myślisz?

Można tak zrobic i wtedy bedzie napis z alt co powinno tam być albo dodać img, które będzie wyswietlane domyślnie, gdy ścieżka będzie błędna albo obrazka w ogóle nie będzie. Mogę zrobić pierwszą lub drugą opcje.

Reply:
Wybrałem opcje, że jest zwracany komponent bez obrazka.